mhs-reference-implementation/mhs/routing/sds_connection_factory.py

```diff
@@ -0,0 +1,31 @@
+import ssl
+import ldap3
+
+
+def build_sds_connection(ldap_address: str) -> ldap3.Connection:
+    """
+    Given an ip address this will return a ldap3 connection object
```

Comment:
Pedantic/minor: do you have to pass an ip address or any address?

Reply:
Looking at the ldap lib it seems like any valid ldap address with be accepted, I'll update to reflect this